hubotjs-environment/secrets.example.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: gh-applets
+type: Opaque
+data:
+  GH_APP_ID: 'YOUR_BASE64_ENCODED_APP_ID'
+  GH_PEM_KEY: 'YOUR_BASE64_ENCODED_APP_SECRET_KEY'
+
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: ghapplets-jira-pr
+type: Opaque
+data:
+  GH_APP_ID: 'YOUR_BASE64_ENCODED_APP_ID'
+  GH_INST_ID: 'YOUR_BASE64_ENCODED_INSTALLATION_ID'
+  GH_PEM_KEY: 'YOUR_BASE64_ENCODED_APP_SECRET_KEY'
```

Comment:
Can we consider offloading these from the config, to an environment variable? This would help mitigate risk in storing secret keys in a repo, by accident (Something I've **never** done 😆 )

Reply:
That’s exactly the idea of secrets! The secrets.yaml file you need to create is gitignored already - this guy here is just a template.